StudyPathTraker/StudyPathTraker/Extensions/UINavigationController+Extensions.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  UINavigationController+Extensions.swift
+//  StudyPathTraker
+//
+//  Created by Rafael Lopez on 5/30/18.
+//  Copyright © 2018 Jerti. All rights reserved.
+//
+
+import UIKit
+
+extension UINavigationController {
```

Comment:
I believe we misunderstood this 😅 I just asked why you was subclassing UINavigationController, not that you change 🙂, I still can't figure out what is the purpose of this addition.

Reply:
Just to configure the navigation from the beginning with a subclass of UINavigationViewController

Reply:
Also customizing the nav bar according to the app design

Reply:
ok got it.

Reply:
I believe the subclass works better. this is kind of hack add this behavior to the base implementation instead of override.